Assets/Scripts/Config/StaticConfig.cs

```diff
@@ -37,9 +39,29 @@ public class HitConfig {
     public float killProbability = 0.9f;
   }
 
+  [Serializable]
+  public class PowerTable {
+    public float IDLE;
+    public float LOW;
+    public float CRUISE;
+    public float MIL;
+    public float MAX;
```

Comment:
might be worth adding comments to what MIL is (and that these are most likely in ascending order)